index.js

```diff
@@ -14,8 +14,8 @@ var defaultTransforms = {
   }
 };
 
-function init(target, transforms) {
-  if (target instanceof Sequelize.Model) {
+function init(Sequelize, target, transforms) {
+  if (Sequelize.Model.isPrototypeOf(target)) {
```

Comment:
Will this work for v3 as well?

Reply:
Good point @sameoldmadness , the first idea was actually not to keep compatibility with Sequelize v3 in v2, but it's actually easy to do if we keep both conditions. I'll just merge this PR and do the fixes myself.